src/utils/constants.ts

```diff
@@ -125,16 +145,24 @@ export const dataTypeKind = {
     boolean: "b",
 };
 
+export const itemPathTextContext = (queryName: string, isSource: boolean) => isSource ? `Section1/${queryName}/Source` : `Section1/${queryName}`;
```

Comment:
this should be URI encode, try to add a query with that name: "ConnectionOnly1 %-<>\^_`{|}$&+,/:;=?@\()#!"